examples-extra/docs_example/src/App.tsx

```diff
@@ -1,10 +1,10 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import viteLogo from "/vite.svg";
```

Comment:
Is there a reason you changed these docs/example files?

Reply:
Fixed